File: ForceConnector/About.vb
Public Class frmAbout
    Private Sub frmAbout_Load(sender As Object, e As EventArgs) Handles Me.Load
        Dim brand As String = "Force.com Connector NG " & ThisAddIn.Ver
        Me.lblBrand.Text = brand
    End Sub

    Private Sub btnClose_Click(sender As Object, e As EventArgs) Handles btnClose.Click
        Me.Close()
    End Sub

End Class

File: ForceConnector/About.cs
using System;

namespace ForceConnector
{
    public partial class frmAbout
    {
        public frmAbout()
        {
            InitializeComponent();
            _btnClose.Name = "btnClose";
        }

        private void frmAbout_Load(object sender, EventArgs e)
        {
            string brand = "Force.com Connector NG " + ThisAddIn.Ver;
            lblBrand.Text = brand;
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }
    }
}